resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/submit/KubernetesClientUtils.scala

```diff
@@ -141,8 +141,8 @@ private[spark] object KubernetesClientUtils extends Logging {
         }
       }
       if (truncatedMap.nonEmpty) {
-        logInfo(s"Spark configuration files loaded from $confDir :" +
-          s" ${truncatedMap.keys.mkString(",")}")
+        logInfo(log"Spark configuration files loaded from ${MDC(DIR, confDir)} : " +
```

Comment:
I'm `not sure` if this is appropriate to call `PATH`, maybe it's more appropriate to call `DIR`?